packages/thrift-server-core/tsconfig.json

```diff
@@ -7,13 +7,15 @@
     "declaration": true,
     "rootDir": "./src",
     "outDir": "./dist",
+    "noEmitOnError": true,
+    "strictNullChecks": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
     "allowSyntheticDefaultImports": true,
+    "allowUnreachableCode": true,
```

Comment:
Just curious, why is this something we'd want to allow? It seems sort of on par with `noUnusedLocals`

Reply:
We shouldn't. I'll fix. I was trying to normalize our tsconfigs and must've copied something wrong